src/pages/releases/[repo].ts

```diff
@@ -1,14 +1,14 @@
-import { getRuntime } from "@astrojs/cloudflare/runtime";
-import { APIRoute } from "astro";
+import type { APIRoute } from "astro";
 
 const Repos = {
     installer: "Vencord/Installer",
     vencord: "Vendicated/Vencord",
     "vencord-desktop": "Vencord/Desktop",
 };
 
-export const get: APIRoute = async ({ params, request }) => {
+export const GET: APIRoute = async ({ params, locals }) => {
     const repo = Repos[params.repo as keyof typeof Repos];
+    const runtime = locals.runtime;
```

Comment:
you will be eaten alive by 27 lions tonight if you don't use destructuring (it's true)

Reply:
it doesnt even work :P